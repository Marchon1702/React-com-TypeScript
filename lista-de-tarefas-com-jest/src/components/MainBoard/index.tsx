import TaskBoard from "../TaskBoard";

export default function MainBoard() {
    return (
        <section className="main-board">
            <header>
                <h1>Lista de Tarefas</h1>
            </header>
            <main>
                <TaskBoard />
            </main>
            <footer>
                Desenvolvido por Matheus Marchon
            </footer>
        </section>
    )
}