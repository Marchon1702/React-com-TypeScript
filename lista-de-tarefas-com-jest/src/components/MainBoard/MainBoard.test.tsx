import { render } from "@testing-library/react";
import MainBoard from ".";

describe("Ações MainBoard", () => {
  test("A MainBoard deve renderizar corretamente", () => {
    const { container } = render(<MainBoard />);
    expect(container).toMatchSnapshot();
  });
});
