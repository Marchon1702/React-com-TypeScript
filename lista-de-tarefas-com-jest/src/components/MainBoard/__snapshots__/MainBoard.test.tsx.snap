// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ações MainBoard A MainBoard deve renderizar corretamente 1`] = `
<div>
  <section
    class="main-board"
  >
    <header>
      <h1>
        Lista de Tarefas
      </h1>
    </header>
    <main>
      <section
        class="task-board"
      >
        <form
          role="form"
        >
          <input
            placeholder="Digite uma nova tarefa"
            required=""
            type="text"
            value=""
          />
          <button>
            Adicionar
          </button>
        </form>
        <ul
          role="unordenedList"
        />
      </section>
    </main>
    <footer>
      Desenvolvido por Matheus Marchon
    </footer>
  </section>
</div>
`;
